### PR TITLE
Rework.

### LINKED ISSUE #3: Use `filesystem` module?
https://docs.ansible.com/ansible/filesystem_module.html

### LINKED ISSUE #4: Maybe use `parted` for partitioning?
``` Shell
parted --script /dev/sdb \
    mklabel gpt \ mkpart primary 1MiB 100%
```

### PR DESCRIPTION
**Pull request by ypid**
- No need to differ between Debian and Ubuntu.
- Use `filesytem` module. Closes #3.
- Use `parted` for partitioning. Closes #4.
- Use `blkid` (UUID) for `/etc/fstab`.

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -1,50 +1,39 @@
...
-  when: disk_group is defined and disk_user is defined
-  tags:
-    - disk
+- name: Create filesystem on the first partition
+  filesystem:
+    dev: '{{ item.disk }}1'
+    force: '{{ item.force|d(omit) }}'
+    fstype: '{{ item.fstype }}'
+    opts: '{{ item.fsopts|d(omit) }}'
+  with_items: disk_additional_disks
```
Comment (MiLk) [inline tasks/main.yml:34]:
``` yaml
tags:
  - disk
```

is missing

**Comment by MiLk** [comment]
@tokyowizard Could you test this PR?

**Comment by ypid** [comment]
Do you know https://github.com/nickjj/rolespec which can be used for integration testing?

**Comment by MiLk** [comment]
Yes, but we are currently using https://github.com/AerisCloud/ansible-role-test for our roles.
Which is another alternative.

**Comment by tokyowizard** [comment]
:+1: LGTM. Works on Debian Wheezy, Jessie, CentOS 6.4 and CentOS 7.1.

**Pull request merged**